src/main/resources/dbchangelog.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                    xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+                    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+    <property name="now" value="UNIX_TIMESTAMP()" dbms="postgresql"/>
```

Comment:
Z takim czymś będziemy mieli problem, bo liquibase działa tak:

    Tworzy sobie tabelkę i dodaje tam wpisy o wszystkich wgranych skryptach.
    Na podstawie zawartości skryptu do wgrania tworzy sobie checksumę.
    Sprawdza czy dla klucza id, author już istnieje jakiś wpis
    3a Jeśli nie to wgrywa nowy skrypt
    3b Jesli tak to porównuje z poprzednią cheksumą: Jesli taka sama to przechodzi do kolejnego skryptu. Jesli nie to zgłasza błąd

Przy takiej konfiguracji jaką zrobiłeś przy każdym uruchamianiu liquibase będzie nowa wartość dla id,author bo id zmienia się przy każdym wywołaniu. Dlatego id musi być z góry ustalone.
To co trzeba zrobić to przy dodawaniu skryptu sprawdzić sobie unix timestamp i dodać go ręcznie.
Unix timestamp dlatego, że jest bardzo małe prawdopodobieństwo, że ktoś akurat w tym samym czasie go sobie sprawdzi, a timestamp daje też nam jakąś chronologię.